### ISSUE TITLE
False positive syntax error FirebaseRulesTokenType.SERVICE_KEYWORD

### ISSUE THREAD

**Issue by malliaridis** [anboralabs/intellij-firebase-highlighter#11]
I installed the plugin in Webstorm with the version 1.1.0-SNAPSHOT and I get one false positive error on valid rules.

I got currently only one false positive syntax error when I write the following valid rules:
```rules
rules_version = '2';

function someFunction() { // Error: "FirebaseRulesTokenType.SERVICE_KEYWORD expected, got 'function' "
  return false;
}

service cloud.firestore {
  ...
}
```

Same error occurs when I write
```rules
rules_version = '2';

// some comment  // Error: "FirebaseRulesTokenType.SERVICE_KEYWORD expected, got '// some comment' "
service cloud.firestore {
  ...
}
```

P.S. As overall however it is what I was looking for. Good job. :)

**Comment by dalgarins** [comment]
Thanks for testing, The changes gonna be available for version: 1.1.1
![Screen Shot 2020-10-28 at 10 42 44 PM](https://user-images.githubusercontent.com/8527174/97522974-0e7bf100-196f-11eb-9320-fba03fe43c52.png)
![Screen Shot 2020-10-28 at 10 43 11 PM](https://user-images.githubusercontent.com/8527174/97522977-10de4b00-196f-11eb-8656-0361eaaab8dc.png)

**Comment by dalgarins** [comment]
![Screen Shot 2020-10-28 at 10 43 11 PM](https://user-images.githubusercontent.com/8527174/97522999-1b98e000-196f-11eb-833e-ac0a70aa2032.png)
